README.MD

```diff
@@ -1 +1,18 @@
-[![Coverage Status](https://coveralls.io/repos/github/AgileVentures/WebsiteOne-FE/badge.svg?branch=develop)](https://coveralls.io/github/AgileVentures/WebsiteOne-FE?branch=develop)
\ No newline at end of file
+[![Coverage Status](https://coveralls.io/repos/github/AgileVentures/WebsiteOne-FE/badge.svg?branch=develop)](https://coveralls.io/github/AgileVentures/WebsiteOne-FE?branch=develop)
+
+# Semantic ui - customize
```

Comment:
Is there a way to automate this, so that each person so not need to do this?

Reply:
Can we move this to a section that is called.
**Rebuild the semantic UI asserts**